src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -124,6 +124,13 @@ StartMetadatSyncToNode(char *nodeNameString, int32 nodePort)
 								escapedNodeName, nodePort)));
 	}
 
+	if (workerNode->groupId == 0)
+	{
+		ereport(WARNING, (errmsg("%s:%d is the coordinator and already contains "
+								 "metadata, skipping.", nodeNameString, nodePort)));
```

Comment:
I think `NOTICE` might be more accurate. Is there a reason to warn the user for this? 

And, minor: `skipping ` -> `skipping to sync the metadata` might be little more useful

Reply:
done